modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PoliciesTable/index.tsx

```diff
@@ -503,8 +507,7 @@ const PoliciesTable = () => {
 						}
 						if (renewalDue === 0) {
 							return 'Due Today';
-						}
-						else {
+						} else {
```

Comment:
Run **afcb** to adjust these lines. If possible remove the else.

Reply:
Done.